### PR TITLE
Added abbr element to acronym

### PR DESCRIPTION
**Pull request by vanillajonathan**
Added `required` attribute to `input` element.

### CODE REVIEW

File: http/drivers.php [hunk 1]
```diff
@@ -58,9 +58,9 @@
 
             <h1>Easy decoder ring</h1>
             <p>Note: this is a beta, it only works for AMD graphics cards (for now).</p>
-            <p>Enter the commercial name of your GPU (e.g. HD 6870):</p>
+            <p>Enter the commercial name of your <abbr title="Graphics processing unit">GPU</abbr> (e.g. HD 6870):</p>
```
Comment (MightyCreak) [inline http/drivers.php]: Thank you! Could you use caps for 'P' and 'U' as well?

Comment (vanillajonathan) [inline http/drivers.php]: Okay, I fixed that. :)

**Comment by MightyCreak** [comment]
Would be better if you could get these modification into just one commit (I'm a bit of a commit history freak ;))

You'll be able to do that with `git rebase -i <your_commit>^` (from the top of my head, so not 100% sure)

**Comment by vanillajonathan** [comment]
Oh, I didn't use the Git command-line tool. I did all this from within GitHub on the web.

**Comment by MightyCreak** [comment]
No worries, I'll take care of it then ;)

**Comment by MightyCreak** [comment]
Done with 0bd10f61121eaad3ad32222a5bb98e74d9b0e952 ;)

**Pull request closed without merging**